sla.md

```diff
@@ -12,21 +12,34 @@ Generally the definitions as made and used in the TOS apply.
 
 - "Monthly Uptime Percentage" is calculated by subtracting from 100% the percentage of minutes during the month in which the Giant Swarm Service, as applicable, was in the state of "GS Unavailable." Monthly Uptime Percentage measurements exclude downtime resulting directly or indirectly from any GS SLA Exclusion (defined below).
 
-- "Shared Cluster" is a system where you are running containers without exclusive hardware and virtual servers to power them but on a shared hardware.
+- “Kubernetes Master” is the controlling unit(s) of your Kubernetes Cluster.
 
-- "Private Cluster" is a system of hardware or virtual servers available in your cluster.
+- “Nodes” are your compute instances of your Kubernetes Cluster.
 
-- "Cluster Storage" is our persistent facility.
+- "Cluster Storage" is our persistent storage facility.
```

Comment:
We decided elsewhere to call this "persistent storage". Could be defined like this:

"Persistent Storage" is our facility to provide you with long-term data storage.